hw-6 (weatherstack)/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="" />
+    <title>Weather</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+      <header>
+        <input type="text" id="city" placeholder="City">
+        <input type="text" id="country" placeholder="Country">
+        <button id="button">Submit</button>
```

Comment:
Would be rather better to wrap it with the `form` tag

Reply:
Why is this better?) Because of the <form> tag, I had problems submitting the form (although I solved them, but I had to increase the amount of code)